__tests__/integration/helpers/testRunner.js

```diff
@@ -88,6 +88,7 @@ const setUpTestRunner = async ({ oneAppLocalPortToUse, oneAppMetricsLocalPortToU
     logLevel: 'silent',
     protocol: 'http',
     hostname: 'localhost',
+    path: '/wd/hub',
```

Comment:
Where does this come from?

Reply:
https://github.com/webdriverio/webdriverio/pull/4907/files

Reply:
I wonder why they decided to do that